### DISCUSSION TITLE
Hacktoberfest 2022 - Repositories we :heart:

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/virtualcoffee.io#624] [Monthly Challenge]
We're moving into open source season at Virtual Coffee, and we can't wait to get some VC-verified repositories as we move into Hacktoberfest! 

Once a repository has been evaluated, drop the results here!

**Comment by AlexVCS** [comment]
https://github.com/weareinreach/InReach

✅ It should have a relevant name.
✅ It should have a description.
The description should clearly describes what the project does and is for.
If your project is participating in [Hacktoberfest](https://hacktoberfest.digitalocean.com/), you should also write the Hacktoberfest topic here. Or you can add the hacktoberfest-accepted label to the accepted pull requests. _I messaged Joe about adding this_
✅ It should have relevant topics.
These topics should highlight its scope, stack, field, etc.
✅ It should have a README file.
The README should link to or describe:
* Project purpose and context
* Installation instructions
✅ A CONTRIBUTIONS/CONTRIBUTING markdown file.
It provides contribution guide and consist of:
* The recommended process for making issues/submitting pull requests.
* Ways to communicate with the maintainers.
✅ A valid open source license.
A license is essential. A project is not an open source if it doesn't have a valid open source license.
✅ A Code of Conduct (COC).
The Code of Conduct (COC) is an excellent indicator of a healthy contributor's environment.
❌ Issue and Pull request templates.
These templates guide potential contributors on making issues and pull requests to the project. They remove a lot of burden from the maintainer and the contributor from guessing the appropriate procedure for these contributions.

**Reply by AlexVCS** [reply]
https://github.com/ClJarvis/Hello_There

✅ It should have a relevant name.
✅ It should have a description.
The description should clearly describes what the project does and is for.
If your project is participating in [Hacktoberfest](https://hacktoberfest.digitalocean.com/), you should also write the Hacktoberfest topic here. Or you can add the hacktoberfest-accepted label to the accepted pull requests.
✅ It should have relevant topics.
These topics should highlight its scope, stack, field, etc. _Has some but could use more_
✅ It should have a README file.
The README should link to or describe:
* Project purpose and context
❌ Installation instructions
✅ A CONTRIBUTIONS/CONTRIBUTING markdown file. _This project uses Contribute.md_
It provides contribution guide and consist of:
* The recommended process for making issues/submitting pull requests.
❌ Ways to communicate with the maintainers.
❌ A valid open source license.
A license is essential. A project is not an open source if it doesn't have a valid open source license.
✅ A Code of Conduct (COC).
The Code of Conduct (COC) is an excellent indicator of a healthy contributor's environment.
❌ Issue and Pull request templates.
These templates guide potential contributors on making issues and pull requests to the project. They remove a lot of burden from the maintainer and the contributor from guessing the appropriate procedure for these contributions.

**Reply by BekahHW** [reply]
@ClJarvis if you need some help updating to get this on our list for Hacktoberfest, please reach out in the monthly-challenge channel.

**Reply by ClJarvis** [reply]
I didn't think I needed Issue and Pull request templates. I thought communication method was stated in the contribute.md. I'd like to all contained to github. There's really nothing to install.  Maybe I need to be more explicit. I'll ask for opinions in slack. 
I do need to write up about tagging an issue so it auto closes. I think that's on the Pull request template I help write for VC last year.

**Reply by AlexVCS** [reply]
From my understanding @ClJarvis you don't have to meet every one of these checkpoints. There is an emphasis put on having a valid open source license, which you could add. Your project is pretty close to being good, add the license and any other things you're able to meet more of the checklist.

**Reply by AlexVCS** [reply]
https://github.com/freeCodeCamp/Developer_Quiz_Site

✅ It should have a relevant name.
✅ It should have a description.
The description should clearly describes what the project does and is for.
If your project is participating in [Hacktoberfest](https://hacktoberfest.digitalocean.com/), you should also write the Hacktoberfest topic here. Or you can add the hacktoberfest-accepted label to the accepted pull requests.
✅ It should have relevant topics.
These topics should highlight its scope, stack, field, etc.
✅ It should have a README file.
The README should link to or describe:
* Project purpose and context
* Installation instructions
✅ A CONTRIBUTIONS/CONTRIBUTING markdown file. _This project has their contributor guidelines external_
It provides contribution guide and consist of:
* The recommended process for making issues/submitting pull requests.
* Ways to communicate with the maintainers.
❌ A valid open source license.
A license is essential. A project is not an open source if it doesn't have a valid open source license.
_freeCodeCamp licenses this themselves_
✅ A Code of Conduct (COC).
The Code of Conduct (COC) is an excellent indicator of a healthy contributor's environment.
❌ Issue and Pull request templates.
These templates guide potential contributors on making issues and pull requests to the project. They remove a lot of burden from the maintainer and the contributor from guessing the appropriate procedure for these contributions.

**Comment by BekahHW** [comment]
- [Deepgram Python SDK](https://github.com/deepgram/python-sdk)
- [Deepgram Node SDK](https://github.com/deepgram/deepgram-node-sdk)
- [Deepgram .NET SDK](https://github.com/deepgram/deepgram-dotnet-sdk)

**Comment by AlexVCS** [comment]
https://github.com/novuhq/novu

✅ It should have a relevant name.

✅ It should have a description.
The description should clearly describes what the project does and is for.
If your project is participating in [Hacktoberfest](https://hacktoberfest.digitalocean.com/), you should also write the Hacktoberfest topic here. Or you can add the hacktoberfest-accepted label to the accepted pull requests.
✅ It should have relevant topics.
These topics should highlight its scope, stack, field, etc.
✅ It should have a README file.
The README should link to or describe:
* Project purpose and context
* Installation instructions
✅ A CONTRIBUTIONS/CONTRIBUTING markdown file.
It provides contribution guide and consist of:
* The recommended process for making issues/submitting pull requests.
* Ways to communicate with the maintainers.
✅ A valid open source license.
A license is essential. A project is not an open source if it doesn't have a valid open source license.
✅ A Code of Conduct (COC).
The Code of Conduct (COC) is an excellent indicator of a healthy contributor's environment.
✅ Issue and Pull request templates.
These templates guide potential contributors on making issues and pull requests to the project. They remove a lot of burden from the maintainer and the contributor from guessing the appropriate procedure for these contributions.

**Comment by AlexVCS** [comment]
https://github.com/stepzen-dev/examples

✅ It should have a relevant name.
✅ It should have a description.
The description should clearly describes what the project does and is for.
If your project is participating in [Hacktoberfest](https://hacktoberfest.digitalocean.com/), you should also write the Hacktoberfest topic here. Or you can add the hacktoberfest-accepted label to the accepted pull requests.
✅ It should have relevant topics.
These topics should highlight its scope, stack, field, etc.
✅ It should have a README file.
The README should link to or describe:
* Project purpose and context
❌ Installation instructions _It's described a bit in contributing.md_
✅ A CONTRIBUTIONS/CONTRIBUTING markdown file.
It provides contribution guide and consist of:
* The recommended process for making issues/submitting pull requests.
* Ways to communicate with the maintainers.
✅ A valid open source license.
A license is essential. A project is not an open source if it doesn't have a valid open source license.
❌ A Code of Conduct (COC).
The Code of Conduct (COC) is an excellent indicator of a healthy contributor's environment.
❌ Issue and Pull request templates.
These templates guide potential contributors on making issues and pull requests to the project. They remove a lot of burden from the maintainer and the contributor from guessing the appropriate procedure for these contributions.

**Comment by rmirville** [comment]
Strapi CMS: https://github.com/strapi/strapi

* [x] It should have a relevant name.
* [x] It should have a description.

> Strapi is the leading open-source headless CMS. It’s 100% JavaScript, fully customizable and developer-first.

* [x] It should have relevant topics.

> nodejs javascript mysql api graphql cms koa rest dashboard content-management customizable koa2 strapi cms-framework jamstack ***hacktoberfest*** content-management-system headless-cms

* [x] It should have a README file. - https://github.com/strapi/strapi#readme
The README should link to or describe:

    * [x] Project purpose and context
    * [x] Installation instructions - https://github.com/strapi/strapi#-installation

* [x] A CONTRIBUTIONS/CONTRIBUTING markdown file. - https://github.com/strapi/strapi/blob/main/CONTRIBUTING.md
It provides contribution guide and consist of:

    * [x] The recommended process for making issues/submitting pull requests. 
    * [x] Ways to communicate with the maintainers. - provided in README: https://github.com/strapi/strapi#community-support

* [x] A valid open source license. (MIT Expat License) - https://github.com/strapi/strapi/blob/main/LICENSE

* [x] A Code of Conduct (COC). - https://github.com/strapi/strapi/blob/main/CODE_OF_CONDUCT.md


* [x] Issue and Pull request templates.
    * [x] Issue - https://github.com/strapi/strapi/blob/main/.github/ISSUE_TEMPLATE/BUG_REPORT.md
    * [x] Pull request - https://github.com/strapi/strapi/blob/main/.github/PULL_REQUEST_TEMPLATE.md

**Comment by BekahHW** [comment]
- [freeCodeCamp Quiz Site](https://github.com/freeCodeCamp/Developer_Quiz_Site)
- [ToolJet](https://www.github.com/ToolJet/Tooljet)

**Comment by sadiejay** [comment]
https://github.com/codethesaurus/codethesaur.us  (website)
https://github.com/codethesaurus/docs  (docs)
